fonts/README.md

```diff
@@ -0,0 +1,15 @@
+# Fonts
+
+Sigma-9 self-hosts several fonts to reduce external depency and minimize `@import`, which are render-blocking resources and potentially costly in terms of performance.
+
+## [Inter](https://github.com/rsms/inter)
+
+Sigma-9's body font. Licensed under [SIL Open Font License 1.1](https://github.com/rsms/inter/blob/master/LICENSE.txt).
+
+ ## [Sans Normalcy](https://github.com/scpwiki/normalcy-sans)
```

Comment:
I edited the suggestion block but you had already committed. Sorry, I made a mistake since mobile, should be:
```suggestion
## [Sans Normalcy](https://github.com/scpwiki/normalcy-sans)
```